Formula/theos.rb

```diff
@@ -0,0 +1,45 @@
+class Theos < Formula
+  desc "Cross-platform tools for developing software for iOS and other platforms"
+  homepage "https://theos.dev"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theos/theos.git", branch: "master"
+
+  stable do
+    url "https://github.com/theos/theos.git",
+      branch:   "master",
+      revision: "e53ec51e92a3716def537c7ff1b5b2a86930700c"
+    version "2.6"
+  end
+
+  keg_only "symlinking Theos makes it unusable"
+
+  depends_on "ldid"
+  depends_on :macos
+  depends_on :xcode
+  depends_on "xz"
+
+  skip_clean "lib", "sdks"
+
+  resource "sdks" do
+    url "https://github.com/theos/sdks.git",
+      branch:   "master",
+      revision: "fd931ca723ee46994ad9f73c2f0929d1ab9732ca"
+  end
+
+  def install
+    prefix.install Dir["*"]
+    (prefix/"sdks").install resource("sdks")
+    rm_rf lib/".keep"
+  end
+
+  def caveats
+    <<~EOS
+      To use Theos, please put the following in your .profile:
+        export THEOS="#{opt_prefix}"
+    EOS
```

Comment:
We should probably use `wrapper_script` to do this for people

Reply:
So just use echo to add this line to .profile/.zshrc/.bashrc?

Reply:
No, use `wrapper_script`

Reply:
Cant find any documentation on what wrapper_script is and how to use it. Do you have a link?

Reply:
https://rubydoc.brew.sh/Pathname.html#write_env_script-instance_method for example

Reply:
Thanks! So theos is not just 1 command/file, there are multiple files which require the THEOS env variable and it would just be a lot easier for user to set that in their .profile rather than us making a wrapper script of almost all executables. What do you think?

Reply:
There is https://rubydoc.brew.sh/Pathname.html#env_script_all_files-instance_method for wrapping all files. People rarely read the caveats so anything that needs can be done automatically should be done automatically

Reply:
It kinda wont work I'm realizing that some files in bin dir are symlinks to files outside of bin dir.